content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -34,24 +34,25 @@ The first step when adding non-Kubernetes services to an Istio mesh is to config
 generate the configuration files that let mesh expansion VMs connect to the mesh. To prepare the
 cluster for mesh expansion, run the following commands on a machine with cluster admin privileges:
 
-1.  Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion=true` at
+1.  Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion.enabled=true` at
     install time with Helm, there are two options for enabling mesh expansion, depending on how you originally installed
     Istio on the cluster:
 
     *   If you installed Istio with Helm and Tiller, run `helm upgrade` with the new option:
 
     {{< text bash >}}
     $ cd install/kubernetes/helm/istio
-    $ helm upgrade --set global.meshExpansion=true istio-system .
+    $ helm upgrade --set global.meshExpansion.enabled=true istio-system .
     $ cd -
     {{< /text >}}
 
     *   If you installed Istio without Helm and Tiller, use `helm template` to update your configuration with the option and
         reapply with `kubectl`:
 
     {{< text bash >}}
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
     $ cd install/kubernetes/helm/istio
-    $ helm template --set global.meshExpansion=true --namespace istio-system . > istio.yaml
+    $ helm template --set global.meshExpansion.enabled=true --namespace istio-system . > istio.yaml
```

Comment:
done.

Reply:
This suggested change is not necessary. The first resource in the `istio.yaml` file is the `istio-system` namespace.
Please remove.

Reply:
Throughout this doc, we don't need to guess which commands are necessary or which commands work. We can run these commands and verify. IMO, verified commands should be the bar we need to meet for adding to documentation.

Reply:
It maybe env or release discrepancy, but I **verified** this on my workstation before taking this suggestion,

```
helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
  --set global.meshExpansion.enabled=true > ./istio.yaml  && ack -i "kind.*namespace" istio.yaml 
```

Reply:
Just to be clear, I was responding to the suggestion that the `istio-system` namespace be explicitly created by the user. That's not necessary since the `istio.yaml` will do that automatically.